src/Analyzers/VisualBasic/CodeFixes/AddExplicitCast/VisualBasicAddExplicitCastCodeFixProvider.vb

```diff
@@ -74,12 +74,20 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.CodeFixes.AddExplicitCast
                         mutablePotentialConversionTypes.Add((spanNode, conversionType))
                     End If
                 Case BC30518, BC30519
-                    Dim invocationNode = spanNode.GetAncestors(Of ExpressionSyntax).FirstOrDefault(
+                    Dim invocationNodeExpression = spanNode.GetAncestors(Of ExpressionSyntax).FirstOrDefault(
+                        Function(node) Not node.ChildNodes.OfType(Of ArgumentListSyntax).IsEmpty())
```

Comment:
Unclear as to what this change is for. 

Not against making the change, Just need clarification on why this is better than the code that was originally there.

Reply:
instead of checking every arbitrary node, you are only looking at invocations.  and because you're only looking at invocatinos, you don't have to enumerate all child nodes looking for argumentlists, you can just directly check if it has an argument list or not.

(the same feedback applies to the check below this for atttributes as well).

thanks!

Reply:
Gotcha! I'll make that change tonight and test it once I'm off work

thanks.